public/js/test/mochitest/examples/README.md

```diff
@@ -0,0 +1,5 @@
+### Test Examples
+
+##### Pages
+* **doc_script-switching-01** - includes two scripts that reference each other. The second function has a debugger.
+* **doc-scripts** - includes three sources, a long source and two sources that reference each other.
```

Comment:
I'd like to start documenting our test examples from the start from the beginning. It is really easy to add more and more examples when there is a perfectly good one already.